### PR TITLE
Complete Official Set of Images & Audio

### PR DESCRIPTION
**Pull request by Pippow**
- Added all missing arts from Official Gwent (incl. Rogue Mage)
- Renamed all unused cards to official ID so it's easily searchable through https://gwent.one/en/cards/
- Uncensored the cards that @Mattia900 had done
- Added every official voiceline to the game, for existing and future cards

Big thanks to @kajtek14102 whose work contributed greatly for the sounds part

### CODE REVIEW

**Comment by The-last-Iris** [comment]
> * Added all missing arts from Official Gwent (incl. Rogue Mage)
> * Renamed all unused cards to official ID so it's easily searchable through https://gwent.one/en/cards/
> * Uncensored the cards that @Mattia900 had done

I have heard that you have finished the new arts and audio frequency,could we add this to the existing client?And I have some question if deck code will lose effectiveness after you renamed all unused cards to official ID

**Comment by Pippow** [comment]
> > * Added all missing arts from Official Gwent (incl. Rogue Mage)
> > * Renamed all unused cards to official ID so it's easily searchable through https://gwent.one/en/cards/
> > * Uncensored the cards that @Mattia900 had done
> 
> I have heard that you have finished the new arts and audio frequency,could we add this to the existing client?And I have some question if deck code will lose effectiveness after you renamed all unused cards to official ID

Hello, I did finish the arts (was a huge work) and I made a working client with it. Currently its only purpose is that is has uncensored arts. I don't think it causes any issue with the deck codes since it's only the arts ID that was changed, but you can ask Neal for confirmation.
On sounds @kajtek14102 was working on it but he is far from finished.
There are quite a lot of changes we would like to do to the client, like adding an autoforfeit if the timer runs out x times in a row or the ability to move decks in the deck builder, but it's beyond my skills at the moment. Maybe TacoCat can help.

**Comment by The-last-Iris** [comment]
> > > * Added all missing arts from Official Gwent (incl. Rogue Mage)
> > > * Renamed all unused cards to official ID so it's easily searchable through https://gwent.one/en/cards/
> > > * Uncensored the cards that @Mattia900 had done
> > 
> > 
> > I have heard that you have finished the new arts and audio frequency,could we add this to the existing client?And I have some question if deck code will lose effectiveness after you renamed all unused cards to official ID
> 
> Hello, I did finish the arts (was a huge work) and I made a working client with it. Currently its only purpose is that is has uncensored arts. I don't think it causes any issue with the deck codes since it's only the arts ID that was changed, but you can ask Neal for confirmation. On sounds @kajtek14102 was working on it but he is far from finished. There are quite a lot of changes we would like to do to the client, like adding an autoforfeit if the timer runs out x times in a row or the ability to move decks in the deck builder, but it's beyond my skills at the moment. Maybe TacoCat can help.

Thanks for your hardworking!So maybe we can update the arts first,and after that we can discuss the Ogroids.

**Comment by Pippow** [comment]
> > > > * Added all missing arts from Official Gwent (incl. Rogue Mage)
> > > > * Renamed all unused cards to official ID so it's easily searchable through https://gwent.one/en/cards/
> > > > * Uncensored the cards that @Mattia900 had done
> > > 
> > > 
> > > I have heard that you have finished the new arts and audio frequency,could we add this to the existing client?And I have some question if deck code will lose effectiveness after you renamed all unused cards to official ID
> > 
> > 
> > Hello, I did finish the arts (was a huge work) and I made a working client with it. Currently its only purpose is that is has uncensored arts. I don't think it causes any issue with the deck codes since it's only the arts ID that was changed, but you can ask Neal for confirmation. On sounds @kajtek14102 was working on it but he is far from finished. There are quite a lot of changes we would like to do to the client, like adding an autoforfeit if the timer runs out x times in a row or the ability to move decks in the deck builder, but it's beyond my skills at the moment. Maybe TacoCat can help.
> 
> Thanks for your hardworking!So maybe we can update the arts first,and after that we can discuss the Ogroids.

Well it will be necessary to implement new cards such as the Ogroids. We can totally merge it now since it's on a separate branch. But if anyone in QQ has any plans to upgrade the client it would be good to release the client upgrades in a single go.

File: src/Cynthia.Card.Unity/src/Cynthia.Unity.Card/Assets/Addressables/Cards/removeD.ipynb
Comment (Pippow) [inline src/Cynthia.Card.Unity/src/Cynthia.Unity.Card/Assets/Addressables/Cards/removeD.ipynb:1]: it's removed in an ulterior commit

**Comment by Pippow** [comment]
@neal2018 Hi, I have added the missing audio to the PR, should I upload the meta descriptions of the new audio?

**Comment by Pippow** [comment]
@neal2018 beyond the upload of the meta descriptions of the new audio, EVERYTHING is done. I consider the PR ready for review

As you can see, the number of files is massive so to facilitate your review, here are a few notes:

- you won't be able to just go to files changed, the number is too large, instead you can go to each commit, where you will be able to see the files changed
- each commit name should be more or less explainatory but please ask for clarifications if needed
- I messed up and the PR is for DIY branch and not "New Client", on the plus side it should be directly mergeable and up to date
- There isn't actual code to review (except Gwentmap changes of a few artIDs - which I checked visually). There should only be new assets, meta files and the Default Local Group.asset that are changed/added
- I can undo the gwentmap changes until the release of the client if needed
- As usual, there isn't any urgency I don't expect you to look into it on a busy Monday ^^ Thanks immensly for the help and I hope it will be a smooth review and release

**Comment by Pippow** [comment]
@The-last-Iris I completed the audio! We can release the new client whenever the new animations are ready and I can start working on ogroids!

**Pull request merged**